src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -228,7 +230,7 @@ public String getMateContig() {
 
     @Override
     public int getMateStart() {
-        if ( mateIsUnmapped() ) {
+        if ( !isPaired() || mateIsUnmapped() ) {
```

Comment:
I'd think we'd want to keep this exploding if called on an unpaired read. It seems less safe to have it return a value that doesn't make sense.